### PR TITLE
feat: Add output image in docker-push action.

### PR DESCRIPTION
**Pull request by shuofan**
## Description

A few sentences describing the overall goals of the pull request's commits.

## Checklist

 - [ ] I made sure to check the compatibility of the erda version statemented in the action's spec.yml.
 - [ ] My change is adequately tested.

### CODE REVIEW

File: actions/docker-push/1.0/internal/main.go [hunk 1]
```diff
@@ -62,20 +66,27 @@ func run() error {
...
 
 	if err != nil {
 		fmt.Fprintf(os.Stdout, "failed to process image: %v\n", err)
 		return err
 	}
 
-	if err := filehelper.CreateFile(filepath.Join(cfg.WorkDir, "pack-result"), string(resultBytes), 0644); err != nil {
+	if err := storeMetaFile(&cfg, image); err != nil {
 		return err
 	}
```
Comment (chengjoey) [inline actions/docker-push/1.0/internal/main.go:83]: use `pkg.NewResultMetaCollector` collect result meta

File: actions/docker-push/1.0/internal/main.go [hunk 2]
```diff
@@ -22,7 +24,8 @@ import (
 var getRegistry = func(image string) string { return strings.Split(image, "/")[0] }
 
 type Conf struct {
-	WorkDir string `env:"WORKDIR"`
+	WorkDir  string `env:"WORKDIR"`
+	MetaFile string `env:"METAFILE"`
```
Comment (chengjoey) [inline actions/docker-push/1.0/internal/main.go:28]: use `envconf.PlatformParams` define platform params

File: actions/docker-push/1.0/internal/main.go [hunk 3]
```diff
@@ -62,20 +66,27 @@ func run() error {
...
-	if err := filehelper.CreateFile(filepath.Join(cfg.WorkDir, "pack-result"), string(resultBytes), 0644); err != nil {
+	if err := storeMetaFile(&cfg, image); err != nil {
 		return err
 	}
+	fmt.Fprintf(os.Stdout, "successfully upload metafile\n")
 
+	if err := filehelper.CreateFile(filepath.Join(cfg.WorkDir, "pack-result"), string(resultBytes), 0644); err != nil {
+		return err
+	}
+	fmt.Fprintf(os.Stdout, "successfully write pack-result\n")
```
Comment (chengjoey) [inline actions/docker-push/1.0/internal/main.go:89]: use logrus is better
```
logrus.SetFormatter(&logrus.TextFormatter{
		DisableColors:    true,
		DisableTimestamp: true,
	})
	logrus.SetOutput(os.Stdout)
```

**Review by chengjoey — APPROVED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**